dir.props

```diff
@@ -127,6 +128,7 @@
   <ItemGroup>
     <ProjectJsonFiles Include="$(SourceDir)**/project.json" />
     <ProjectJsonFiles Include="$(MSBuildThisFileDirectory)pkg/**/project.json" />
+    <ProjectJsonFiles Condition="'$(BuildTestsAgainstPackages)' == 'true'" Include="$(GeneratedProjectJsonDir)/**/project.json"  />
```

Comment:
Two nits:
1. I'd prefer "GeneratedProjectJsonDir" itself followed standard MSBuild convention and ended with a '/', but I understand that I'm the one who authored it without the '/'. :/
2. Fix spacing before the closing "/>" on this line.

Otherwise, LGTM.